src/test/java/kitchenpos/application/OrderTableServiceTest.kt

```diff
@@ -0,0 +1,129 @@
+package kitchenpos.application
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import kitchenpos.domain.FakeOrderRepository
+import kitchenpos.domain.FakeOrderTableRepository
+import kitchenpos.domain.Order
+import kitchenpos.domain.OrderStatus
+import kitchenpos.domain.OrderTable
+import java.util.UUID
+
+private val orderTableRepository = FakeOrderTableRepository()
+private val orderRepository = FakeOrderRepository()
+
+private val orderTableService = OrderTableService(orderTableRepository, orderRepository)
+
+private fun createOrderTable(name: String? = "테이블") =
+    OrderTable().apply {
+        id = UUID.randomUUID()
+        this.name = name
+        numberOfGuests = 0
+    }
+
+class OrderTableServiceTest : BehaviorSpec({
+    given("주문 테이블을 생성할 때") {
+        `when`("입력 값이 정상이면") {
+            val newTable = createOrderTable()
+
+            then("주문 테이블이 생성된다.") {
+                with(orderTableService.create(newTable)) {
+                    id shouldNotBe null
+                    name shouldBe "테이블"
+                    numberOfGuests shouldBe 0
+                }
+            }
+        }
+
+        `when`("주문 테이블의 이름이 null이면") {
+            val newTable = createOrderTable(name = null)
+
+            then("예외가 발생한다.") {
+                shouldThrow<IllegalArgumentException> {
+                    orderTableService.create(newTable)
+                }
+            }
+        }
+
+        `when`("주문 테이블의 이름이 빈 문자열이면") {
+            val newTable = createOrderTable(name = "")
+
+            then("예외가 발생한다.") {
+                shouldThrow<IllegalArgumentException> {
+                    orderTableService.create(newTable)
+                }
+            }
+        }
+    }
+
+    given("주문 테이블에 손님이 않을 떄") {
```

Comment:
중요한 부분은 아니나 오타가 있네요!

```suggestion
    given("주문 테이블에 손님이 앉을 때") {
```